plugins/modules/ec2_group.py

```diff
@@ -172,6 +192,18 @@
             - The end of the range of ports that traffic is going to.
             - A value can be between C(0) to C(65535).
             - A value of C(-1) indicates all ports (only supported when I(proto=icmp)).
+        icmp_type:
+          type: int
+          description:
+          - When using C(proto: icmp) or C(proto: icmpv6), allows you to specify
+          - the ICMP type to use. The option is mutually exclusive with from_port.
+          - A value of C(-1) indicates all ICMP types.
+        icmp_code:
+          type: int
+          description:
+          - When using C(proto: icmp) or C(proto: icmpv6), allows you to specify
+          - the ICMP code to use. The option is mutually exclusive with C(to_port).
+          - A value of C(-1) indicates all ICMP codes.
```

Comment:
I guess these are redundant since you have already specified these parameters in the documentation.

Reply:
I wasn't sure whether the documentation section in the code was used elsewhere, such as via the `ansible-docs` commands. So I should remove them entirely?